docs/dev/api/accounts.md

```diff
@@ -12,7 +12,7 @@ The Accounts API exposes the following methods related to individual and team ac
 
```

Comment:
The description above (on line 11) could be updated as well:

**from:** _The Accounts API exposes the following **methods** related to individual and team account configuration and monitoring._  
**to:** _The Accounts API exposes the following **endpoints** related to individual and team account configuration and monitoring._